Lib/fontTools/feaLib/builder.py

```diff
@@ -576,7 +577,11 @@ def buildLookups_(self, tag):
                 continue
             lookup.lookup_index = len(lookups)
             lookups.append(lookup)
-        return [l.build() for l in lookups]
+        try:
+            otLookups = [l.build() for l in lookups]
+        except OpentypeLibError as e:
+            raise FeatureLibError("Failed to build",()) from e
```

Comment:
I think `location=None` is better than an empty () tuple here. Also, space after comma.

```suggestion
            raise FeatureLibError("Failed to build", location=None) from e
```